Feat/better intro (#27)
cicd-lectures/slides: content/chapitres/welcome.adoc

[{invert}]
= Bonjour !

[{invert}]
== Damien DUPORTAL

> * Freelancer
> * Consultant @ link:https://www.ovh.com/fr/[OVHCloud]

Reviewer: ```suggestion
* Señor 🌮 Software Engineer chez https://www.cloudbees.com[CloudBees] sur le projet Jenkins 👨🏻‍⚖️
* Freelancer
```

* Me contacter :

> ** +++<span class="fa fa-envelope"></span>+++ damien.duportal+pro <chez> gmail.com

Reviewer: ```suggestion
** +++<span class="fa fa-envelope"></span>+++ damien.duportal <chez> gmail.com
```

** link:https://www.linkedin.com/in/damien-duportal-ab70b524/[+++<span class="fab fa-linkedin"></span>+++ Damien Duportal,window=_blank]
** link:https://twitter.com/DamienDuportal[+++<span class="fab fa-twitter"></span>+++ @DamienDuportal,window=_blank]

[{invert}]
== Julien LEVESY

* Senior Software Engineer @ link:https://www.upfluence.com/fr/[Upfluence]

* Me contacter :
** +++<span class="fa fa-envelope"></span>+++ jlevesy <chez> gmail.com
** link:https://www.linkedin.com/in/julien-levesy-5b80ab149/[+++<span class="fab fa-linkedin"></span>+++ Julien Levesy,window=_blank]
** link:https://twitter.com/jlevesy[+++<span class="fab fa-twitter"></span>+++ @jlevesy,window=_blank]
** link:https://github.com/jlevesy[+++<span class="fab fa-github"></span>+++ @jlevesy,window=_blank]

== Et vous ?

> image::youreup.gif[width="400"]

Reviewer: ```suggestion
image::youreup.gif[width="800"]
```

== A propos du cours

> * On s'est adaptés à la situation et à voulu faire quelque chose d'interactif

Reviewer: ```suggestion
* On a essayé de s'adapter à la situation et avons essayé de faire quelque chose d'interactif
```

* Il y aura donc une alternance de théorie et de pratique
* C'est la première fois qu'on le donne, il risque d'y avoir des soucis, be kind :-)
[.small]

> ** N'hésitez pas à ouvrir des PRs si vous en voyez (wink wink)

Reviewer: ```suggestion
** N'hésitez pas à ouvrir des PRs si vous en voyez link:https://github.com/cicd-lectures/slides/pulls[ici] (😉 wink wink)
```

> == Outils necessaires

Reviewer: ```suggestion
== Outils Nécessaires
```

> * Un navigateur récent

Reviewer: ```suggestion
* Un navigateur récent (et décent)
```

> * Un compte link:https://github.com[GitHub]

Reviewer: ```suggestion
* Un compte link:https://github.com[GitHub,window="_blank"]
```

> * Un compte link:https://gitpod.io[GitPod.io], notre environement de développement

Reviewer: ```suggestion
* Un compte link:https://gitpod.io[GitPod.io,window="_blank"], notre environement de développement
```
